guides/common/assembly_configuring-provisioning-resources.adoc

```diff
@@ -7,6 +7,16 @@ include::modules/proc_setting-the-provisioning-context.adoc[leveloffset=+1]
 include::modules/proc_creating-operating-systems.adoc[leveloffset=+1]
 
 ifndef::satellite[]
+:os_codename: Bullseye
```

Comment:
At least facter reports it in lower case so I'd expect:
```suggestion
:os_codename: bullseye
```

Reply:
Applied.